### DISCUSSION TITLE
Is there any plans to update codegen v2 manifest?

### DISCUSSION THREAD

**Discussion by Rimantovas** [trustwallet/wallet-core#3912] [Q&A]
Hey, I started looking at your swift codegen tool written in rust. I migrated most of it to generate dart code, but some of the manifest yaml files are behind. Is there any plans to update them?

**Comment by HonestCodeWasTaken** [comment]
Curious about this as well! any updates?

**Comment by satoshiotomakan** [comment]
Hi @Rimantovas, @HonestCodeWasTaken, we suspended the work on codegen-v2.
If you're able, please open a PR to update the manifest files to support Dart lang
